providers/gcp/config/config.go

```diff
@@ -50,6 +50,7 @@ Examples with the GCP project configured:
 				resources.DiscoveryGkeClusters,
 				resources.DiscoveryStorageBuckets,
 				resources.DiscoveryBigQueryDatasets,
+				resources.DiscoverCloudSQLs,
```

Comment:
i would have expected to see 5 new things here? one for each of the items you listed in the description?

Reply:
Yup. Will update this too.